applications/fastapi-bootcamp/templates/ingress.yaml

```diff
@@ -22,7 +21,7 @@ template:
       - host: {{ required "global.host must be set" .Values.global.host | quote }}
         http:
           paths:
-            - path: {{ .Values.config.pathPrefix | quote }}
+            - path: "/fastapi-bootcamp"
```

Comment:
Sorry, Jonathan.  Take it up with Russ.

Reply:
...that's an odd way of saying "hey, I think there's a bug in the starter." I'll go fix the starter; I added the config setting but then forgot to use it.

Reply:
Oh, the history behind that is that on the first iteration of that, I had "/fastapi-bootcamp", and Jonathan caught it.  I figured this was a disagreement over how opinionated we wanted to be.